### ISSUE TITLE
Error name 'unicode' is not defined when using python3

### ISSUE THREAD

**Issue by piyush-saurabh** [mrts/burp-suite-http-proxy-history-converter#10]
While trying to generate csv, I got the error
python convert-burp-suite-http-proxy-history-to-csv.py proxy-history.txt --format csv --csv-delimiter ','

```bash
line 211, in <module>
    main()
  File "/Users/pisaurabh/Documents/work/tools-scripts/burp-suite-http-proxy-history-converter/convert-burp-suite-http-proxy-history-to-csv.py", line 21, in main
    set_csv_delimiter(args.csv_delimiter)
  File "/Users/pisaurabh/Documents/work/tools-scripts/burp-suite-http-proxy-history-converter/convert-burp-suite-http-proxy-history-to-csv.py", line 98, in set_csv_delimiter
    _g_csv_delimiter = unicode(csv_delimiter)
NameError: name 'unicode' is not defined
```

I am using Python 3.9.7 and running the script in python virtual environment on updated Intel Mac.

I solved the issue by adding the following snippet before main
```python
if sys.version_info[0] >= 3:
    unicode = str
```
Ref: https://stackoverflow.com/questions/19877306/nameerror-global-name-unicode-is-not-defined-in-python-3

I request the maintainer(s) to add the following check

**Comment by mrts** [comment]
Hi @piyush-saurabh! Thank you for the suggestion! Can you send a pull request please?

**Comment by mrts** [comment]
Fixed with #11
